warehouse/templates/pages/help.html

```diff
@@ -13,53 +13,76 @@
 -#}
 {% extends "base.html" %}
 
+{% macro code_of_conduct() %}
+  <div class="callout-block">
+    <p><strong>Please note:</strong> All users submitting feedback, reporting issues or contributing to Warehouse are expected to follow the <a href="//pypa.io/en/latest/code-of-conduct/">PyPA Code of Conduct</a>.</p>
+  </div>
+{% endmacro %}
+
 {% block title %}Help{% endblock %}
 
 {% block content %}
   <section class="horizontal-section">
     <div class="narrow-container">
       <h2>Common Questions</h2>
 
-      <h3 id="installing" class="common-question">What is a package and how do I install one?</h3>
 
-      <p>A 'package' in the context of PyPI is a bundle of software to be installed.</p>
-      <p>Packages on PyPI are developed and shared by other members of the Python community so that you can use them in your own projects.</p>
+      <h3 id="packages" class="common-question">What's a package, project, or release?</h3>
+
+      <p>We use a number of terms to describe software available on PyPI, like <em>project</em>, <em>release</em>, <em>file</em>, and <em>package</em>. Sometimes those terms are confusing because they're used to describe different things in other contexts. Here's how we use them on PyPI:</p>
+
+      <p>A <em>project</em> on PyPI is the name of a collection of releases and files, and information about them. Projects on PyPI are made and shared by other members of the Python community so that you can use them.</p>
+
+      <p>A <em>release</em> on PyPI is a specific version of a project. For example, the <a href="//pypi.io/project/requests/">requests</a> project has many releases, like <em>requests 2.10</em> and <em>requests 1.2.1</em>. A release consists of one or more <em>files</em>.</p>
+
```

Comment:
You can do ``{{ request.route_path('packaging.project', name='requests') }}`` to autogenerate a thing like ``/project/requests/``. This will automatically update if we move URLs around, *but* it requires you to know the name of a package that exists on both ``test.pypi.io`` and ``pypi.io`` else it'll link to a page that 404s.